system-tests/project-fixtures/react/cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'cypress'
```

Comment:
Can we make this as minimal as we can? This is a javascript project so we could go with a js config and drop the types.

Reply:
I had been using this as a place to try out the `DefineDevServerConfig`, also since users can define either it shouldn't make a difference. Was also thinking we might want to add `ts` / `tsx` files into this package at some point.